### PR TITLE
[release-4.10] Bug 2054562: Remove icon field from the ART operator bundle

### PR DESCRIPTION
**Pull request by pliurh**

### CODE REVIEW

File: manifests/stable/dpu-network-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -11,6 +11,7 @@ metadata:
             "name": "ovnkubeconfig-sample"
           },
           "spec": {
+            "kubeConfigFile": "tenant-cluster-1-kubeconf",
```
Comment (bn222) [inline manifests/stable/dpu-network-operator.clusterserviceversion.yaml:14]: Is this required?

Comment (pliurh) [reply manifests/stable/dpu-network-operator.clusterserviceversion.yaml:14]: yes

**Pull request merged**